ui/.gitignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
+node_modules
+/dist
+
+
+# local env files
+.env.local
```

Comment:
Are you actually using those? I don't see a `.env`

Reply:
Done